### PR TITLE
[orchestrator] move added node tag name

### PR DESCRIPTION
**Pull request by nammn**
### What does this PR do?

Move added `tag` name from `nodes` from `node_role` to `kube_node_role`

### Motivation

The host tagger already tags the host. We still want to add the tags ourselves to be safe. 
But in case they are the same we should give them the same name out of consistency.

### Additional Notes

Anything else we should know when reviewing?

### Describe your test plan

Write there any instructions and details you may have to test your PR.

### CODE REVIEW

File: releasenotes/notes/fix-tag-name-9f31ba1dce867c81.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+fixes:
+  - |
+    Change added node tag name from node_role to kube_node_role.
```
Comment (mfpierre) [inline releasenotes/notes/fix-tag-name-9f31ba1dce867c81.yaml:11]: Not sure I properly understand this changelog 😅

Comment (nammn) [reply releasenotes/notes/fix-tag-name-9f31ba1dce867c81.yaml:11]: Uh, yeah that does sound bad 😓 . Let me rephrase it. 
`Rename node role tag from "node_role" to "kube_node_role"`

**Review by xlucas — APPROVED** [review]

File: releasenotes/notes/fix-tag-name-9f31ba1dce867c81.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+fixes:
+  - |
+    Rename node role tag from "node_role" to "kube_node_role" in orchestrator_explorer collection.
```
Comment (ahmed-mez) [inline releasenotes/notes/fix-tag-name-9f31ba1dce867c81.yaml:11]: Let's move it under `releasenotes-dca`

Comment (nammn) [reply releasenotes/notes/fix-tag-name-9f31ba1dce867c81.yaml:11]: Ah, yes thanks 👍
